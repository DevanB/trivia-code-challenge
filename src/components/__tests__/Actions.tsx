import {render} from '@testing-library/react'
import * as React from 'react'
import {Actions} from '../Actions'

describe('<Actions/>', () => {
  it('should render without crashing', () => {
    const {container} = render(<Actions />)
    expect(container).toMatchSnapshot()
  })
})
