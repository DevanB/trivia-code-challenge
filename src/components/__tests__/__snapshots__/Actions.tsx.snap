// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Actions/> should render without crashing 1`] = `
<div>
  <div
    class="Actions-sc-112osew-0 hAQIt"
  />
</div>
`;
